### ISSUE TITLE
Got error 500 when try to view contact, which has wrong email address based on bounce from mail server

### ISSUE THREAD

**Issue by magician7** [mautic/mautic#13530]
### Mautic Version

5.0.x series

### Way of installing

I installed with composer using https://github.com/mautic/recommended-project

### PHP version

8.1.27

### What browsers are you seeing the problem on?

Not relevant

### What happened?

return error 500 when try to view contact, which has wrong email, which was detected based on bounce

### How can we reproduce this issue?

1. Configure and set Custom return path (bounce) address and Monitored Inbox Settings
2. Add to list contact with wrong email (non exist). Wait a while until server respond there was a wrong email.
3. Run  mautic:email:fetch
4. Go to dashboard and contacts and try to view this wrong email

### Relevant log output

```shell
Mautic log: mautic.CRITICAL: Uncaught PHP Exception Twig\Error\RuntimeError: "Key "datetime" for array with keys "0" does not exist." at /home/szczescie/mautic_szczescie/docroot/app/bundles/EmailBundle/Resources/views/SubscribedEvents/Timeline/index.html.twig line 49 {"exception":"[object] (Twig\\Error\\RuntimeError(code: 0): Key \"datetime\" for array with keys \"0\" does not exist. at /home/szczescie/mautic_szczescie/docroot/app/bundles/EmailBundle/Resources/views/SubscribedEvents/Timeline/index.html.twig:49)"} {"hostname":"server.webdragon.pl","pid":9649}

Probably there is relevant:
In database in email_stats for this "wrong" email in column open_details there is:
a:1:{s:7:"bounces";a:1:{i:0;a:4:{s:8:"datetime";s:19:"2024-03-08 09:56:04";s:6:"reason";s:7:"unknown";s:4:"code";s:4:"0123";s:4:"type";s:4:"hard";}}}
```

### Code of Conduct

- [X] I confirm that I have read and agree to follow this project's Code of Conduct

<br /><hr>
Care about this issue? Want to get it resolved sooner? If you are a <a href='https://www.mautic.org/become-a-member-of-mautic'>member of Mautic</a>, you can add some funds to the <a href='https://opencollective.com/mautic/projects/bounties'>Bounties Project</a> so that the person who completes this task can claim those funds once it is merged by a member of the core team! Read the docs <a href='https://contribute.mautic.org/product-team/mautic-bounty-programme'>here.</a>

**Comment by magician7** [comment]
there is a typo in the code. I don't know how to submit it to the repository
In app/bundles/EmailBundle/Resources/views/SubscribedEvents/Timeline/index.html.twig line 43 is:
` {% if 'bouces' != key %}`
should be:
` {% if 'bounces' != key %}`
I checked. It works

**Comment by magician7** [comment]
I just saw pull request #13358

**Comment by RCheesley** [comment]
Hi @magician7 - please test that PR and see if it fixes your issue as we need two testers and a code review to release the fix - https://mau.tc/tester has all the information. You can test locally or use Gitpod to test in the browser.

**Comment by magician7** [comment]
@RCheesley I think I done this now ;-) My first time.

**Comment by RCheesley** [comment]
Wahoo, awesome thanks so much for testing @magician7! 🚀

**Comment by RCheesley** [comment]
@all-contributors please add @magician7 for bugs

**Comment by pjrobertson** [comment]
Just came across this bug. For anybody wanting to do a patch fix for this on Mautic 5.0.4, here's what you need to do in terminal :)

```bash
curl https://github.com/mautic/mautic/commit/700a5b260c858aff1a1cb94f70f4a82b053d4a4b.patch | patch -p1
```

**Comment by escopecz** [comment]
Thanks! Just to add, this bug fix will be released in Mautic 5.1.0.
